services/motion/motion_test.go

```diff
@@ -1242,17 +1243,18 @@ func TestMoveOnMapReq(t *testing.T) {
 
 	validMoveOnMapReq := MoveOnMapReq{
 		ComponentName: myBase,
-		Destination:   spatialmath.NewZeroPose(),
+		Destination:   spatialmath.NewPoseFromPoint(r3.Vector{2700, 0, 0}),
 		SlamName:      mySlam,
 		MotionCfg:     motionCfg,
+		Obstacles:     []spatialmath.Geometry{},
 		Extra:         map[string]interface{}{},
 	}
 
 	// RSDK-6444
 	//nolint:staticcheck
 	validPbMoveOnMapNewRequest := &pb.MoveOnMapNewRequest{
 		Name:                "bloop",
-		Destination:         spatialmath.PoseToProtobuf(spatialmath.NewZeroPose()),
+		Destination:         spatialmath.PoseToProtobuf(spatialmath.NewPoseFromPoint(r3.Vector{2700, 0, 0})),
```

Comment:
I don't think you need to change this. It is for moveonmapnew which is being deleted